boards/arm/nucleo_l433rc/Kconfig.defconfig

```diff
@@ -0,0 +1,19 @@
+# STM32L433RC Nucleo board configuration
+#
+# Copyright (c) 2019 Libre Solar Technologies GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NUCLEO_L433RC
+
+config BOARD
+	default "nucleo_l433rc"
+
+config PWM_STM32_2
```

Comment:
You'll have to watch https://github.com/zephyrproject-rtos/zephyr/pull/24860 for upcoming changes on this part.